_layouts/eip.html

```diff
@@ -41,4 +41,11 @@ <h1 class="page-heading">
 
   {{ content }}
 
+  <h2>Citation</h2>
+  <p>Please cite this document as:</p>
+  {% comment %}
+  IEEE specification for reference formatting:
+  https://ieee-dataport.org/sites/default/files/analysis/27/IEEE%20Citation%20Guidelines.pdf
+  {% endcomment %}
+  <p>{% include authorlist.html authors=page.author %}, "EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}{% if page.status == "Draft" or page.status = "Last Call" %} [DRAFT]{% endif %}," <em>Ethereum Improvement Proposals</em>, no. {{ page.eip | xml_escape }}, {{ page.created | date: "%B %Y" }}. [Online serial]. Available: https://eips.ethereum.org/EIPS/eip-{{ page.eip | xml_escape }}.</p>
```

Comment:
CI seems to be unhappy with this:
```
{% if page.status == "Draft" or page.status = "Last Call" %}
```

Reply:
Thank you, corrected at 34d4dd60e0a9891708f65f26951ff005382054cd